plugins/admin/pkg/command/registry/registry.go

```diff
@@ -19,7 +19,35 @@ import (
 	"knative.dev/client-contrib/plugins/admin/pkg"
 )
 
-// privateRegistryCmd represents the privateRegistry command
+var (
+	// AdminRegistryCmdName is used in the labels to mark the resource this command created
+	AdminRegistryCmdName = "kn-admin-registry"
+	// DockerJSONName is used to represent ".dockerconfigjson" in secret data
+	DockerJSONName = ".dockerconfigjson"
+)
+
+// AdminRegistryLabels is a set of labels which will add to the registry resources to indicate
+// that these resources are managed by admin registry command.
+var AdminRegistryLabels = map[string]string{
+	pkg.LabelManagedBy: AdminRegistryCmdName,
+}
+
+// Registry contains data for secret creation
+type Registry struct {
+	Auths Auths `json:"auths"`
+}
+
+// DockerCred contains actual credentials which are used to pull images
+type DockerCred struct {
+	Username string `json:"Username"`
```

Comment:
Can this type be private? That is `dockerCred` instead of `DockerCred`? If only used in this package then make it private.

Reply:
Fixed it by changing back to registryCred as private. This should cover both `the name starts with package name when import` and the image type problems.